packages/ses/types.d.ts

```diff
@@ -42,16 +42,12 @@ export interface RepairOptions {
     // deprecated
     | 'safeEval'
     | 'unsafeEval'
-    | 'noEval';
+    | 'noEval'
+    | 'unsafe-no-direct';
```

Comment:
Put this above the deprecated line, with its kabob-case siblings. Check the other occurrences for similar implicit classification.